packages/blob/src/index.ts

```diff
@@ -122,6 +116,96 @@ export async function put(
   return blobResult;
 }
 
+// vercelBlob.clientPut()
+// This is a wrapper that will fetch the client token for you and then upload the file
+export interface ClientPutCommandOptions extends BlobCommandOptions {
+  access: 'public';
+  contentType?: string;
+  handleClientUploadUrl: string;
+}
+
+export async function clientPut(
```

Comment:
can we try to create a `client/index.ts` and re-export only this file?
I know we said it's fine to expose the `clientPut` function, but think that it would be harder to move it once we'll release the public beta

Reply:
When (= why) would we need to move some methods to /client?

As of today, I am unsure if there will be a time when we will need to move clientPut() to a different import path: all methods of the blob package can be called on the frontend. For example, if you wanted to allow list/delete/head to be called from the frontend, enforcing a specific prefix for which paths can be listed (think: per-user/per-x blob folder).

Reply:
I moved methods to client when they are related to client uploads in general